src/maintainer/adding_pkgs.rst

```diff
@@ -111,6 +113,50 @@ The maintainer's job is to:
 - Answer eventual questions about the package on the feedstock issue tracker.
 
 
+Adding multiple packages at once
+--------------------------------
+
+If you would like to add more than one related packages, they can be added to staged-recipes in a single pull request (in separate directories). If the packages are interdependent (i.e. one package being added lists one or more of the other packages being added as a requirement), conda-build will be able to locate the dependencies that are only present within staged-recipes. Using a single pull request cuts down on the work load for reviewers and allows you to quickly get packages set up without waiting for each package in a dependency chain to be reviewed, built, and added to the conda-forge channel before starting the process over with the next recipe in the chain.
```

Comment:
I would argue that a single pull request increases the work load because you have to review multiple recipes at once. You should add a line that discourages this because for packages with inter-dependencies, after the feedstocks are created, the builds will fail because the dependency is missing.

Reply:
@isuruf we have 6 conda-forge core members here ar SciPy working on this documentation sprint and we agreed that:

- it is easier for the reviewer and the submitter to have everything in a single PR b/c most of the packages are related and that way the same reviewer is responsible for the looking at them all;
- the race condition during the feedstock creation is a small price to pay for that and, as long as that is clear in the docs and we tell them to restart the CIs, it should be OK.

Reply:
> the race condition during the feedstock creation is a small price to pay for that and, as long as that is clear in the docs and we tell them to restart the CIs, it should be OK.

Sure. If it's clear that the maintainer has to do that and there are clear instructions on how to do that on Azure, I'm fine.